### PR TITLE
Non-IDV bidder sees copy in the "registration pending" modals about identity verification

### PR DESCRIPTION
**Pull request by yuki24**
addresses https://artsyproduct.atlassian.net/browse/AUCT-951 and https://artsyproduct.atlassian.net/browse/AUCT-953.

## Todos

 * [x] Fix type errors
 * [x] Add more test coverage
    * [x] `src/lib/Components/Bidding/Screens/Registration.tsx`
    * [x] `src/lib/Components/Bidding/Screens/RegistrationResult.tsx`
   
### Registration Pending on iPhone X

AUCT-953

### Other screens on iPhone SE

Screen Shot 2020-04-03 at 12 10 40 PM

### CODE REVIEW

File: src/lib/utils/renderMarkdown.tsx [hunk 1]
```diff
@@ -209,6 +203,7 @@ export function defaultRules(modal: boolean = false): ParserRules {
     hr: {
       react: () => <Separator mb={2}></Separator>,
     },
+    ...roleOverrides
```
Comment (yuki24) [inline src/lib/utils/renderMarkdown.tsx:206]: @ds300 I wasn't sure if this is a good way to override the default Markdown rules. Do you have any other solutions? Let me know if there's anything better.

Comment (ds300) [reply src/lib/utils/renderMarkdown.tsx:208]: This seems good to me, nice job :+1:

File: src/lib/Components/Bidding/Screens/RegistrationResult.tsx [hunk 2]
```diff
@@ -53,6 +67,19 @@ const registrationNetworkErrorMessage = {
   description: "Please\ncheck your internet connection\nand try again.",
 }
 
+const markdownRules = defaultRules(false, {
+  paragraph: {
+    match: blockRegex(/^((?:[^\n]|\n(?! *\n))+)(?:\n *)/),
```
Comment (erikdstock) [inline src/lib/Components/Bidding/Screens/RegistrationResult.tsx:72]: is this matching a double-newline?

Comment (yuki24) [reply src/lib/Components/Bidding/Screens/RegistrationResult.tsx:72]: I think this is matching a single-newline.

**Comment by yuki24** [comment]
I have seen a bunch of test failures locally due to the RN version upgrade. Not sure what the CI thinks but marking this as WIP for now.

File: src/lib/Components/Bidding/Screens/RegistrationResult.tsx (hunk 2 above)
Comment (yuki24) [reply src/lib/Components/Bidding/Screens/RegistrationResult.tsx:72]: Well, actually, I was wrong. This is probably matching a double-newline.

**Pull request merged**